library/kotlin/src/io/envoyproxy/envoymobile/EnvoyBuilder.kt

```diff
@@ -80,7 +71,12 @@ open class EnvoyBuilder internal constructor(
    * @return A new instance of Envoy.
    */
   fun build(): Envoy {
-    return Envoy(engineType(), resolvedYAML(), logLevel)
+    val configurationYAML = configYAML
```

Comment:
Not sure this really provides any benefit 🤷‍♂

Reply:
It's a variable and the class isn't thread safe so the field could be changed between the lines of code. This way at the start of the method call we are guaranteed to have the correct configuration yaml...

Basically, I get it's silly but the IDE was complaining to me so I changed it :(